fedimint-bin-tests/src/main.rs

```diff
@@ -527,7 +527,237 @@ struct DevFed {
     esplora: Esplora,
 }
 
+fn env_var(var: String) -> Result<Option<String>, std::env::VarError> {
+    match env::var_os(&var) {
+        None => Ok(None),
+        Some(value) => match value.clone().into_string() {
+            Ok(s) => Ok(Some(s)),
+            Err(_) => Err(std::env::VarError::NotUnicode(value)),
+        },
+    }
+}
+
+macro_rules! env {
+    ($name:ident { $( $var:ident = $value:expr ),* $(,)? }) => {
+        struct $name {
+            $( $var: String, )*
+        }
+
+        impl $name {
+            fn init() -> Result<Self, std::env::VarError> {
+                Ok(Self {
+                    $( $var: env_var(stringify!($var).to_owned())?.unwrap_or_else(|| $value.to_owned()), )*
```

Comment:
bind these variables to local variables, so you can use previously defined variables here